### PR TITLE
Add bulk search support for CPEs #808

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

### CODE REVIEW

File: vulnerabilities/tests/test_fix_api.py [hunk 1]
```diff
@@ -270,3 +270,37 @@ def test_api_response(self):
...
+    def test_api_response_with_one_vulnerability(self):
+        request_body = {
+            "cpes": self.cpes,
+        }
+        response = self.client.post(
+            "/api/cpes/bulk_search",
+            data=request_body,
+            content_type="application/json",
+        ).json()
+        assert len(response) == 1
```
Comment (tdruez) [inline vulnerabilities/tests/test_fix_api.py:306]: You need to test the content of the response. The current assertion is too weak and if the content of the response changes this test may not catch it.

**Review by tdruez — CHANGES_REQUESTED** [review]
The unit test needs some improvements.
Also, I think we are missing a changelog entry.

File: vulnerabilities/tests/test_fix_api.py (hunk 1 above)
Comment (TG1999) [reply vulnerabilities/tests/test_fix_api.py:306]: Improved assertions

File: vulnerabilities/tests/test_fix_api.py [hunk 2]
```diff
@@ -270,3 +270,40 @@ def test_api_response(self):
...
+        ]
+        self.cpes = cpes
+        vuln = Vulnerability.objects.create(summary="test")
+        for cpe in cpes:
+            ref = VulnerabilityReference.objects.create(reference_id=cpe)
+            VulnerabilityRelatedReference.objects.create(reference=ref, vulnerability=vuln)
+
+    def test_api_response_with_one_vulnerability(self):
+        request_body = {
+            "cpes": self.cpes,
```
Comment (tdruez) [inline vulnerabilities/tests/test_fix_api.py:299]: A better approach would be to only request a few of the existing cpes along some non existing ones.
Testing on everything is risky as the expecting results can be the same as not filtering at all.

**Review by tdruez — APPROVED** [review]

**Pull request merged**